units/github_status_api.go

```diff
@@ -25,6 +25,7 @@ const (
 	githubStatusUpdateJobName = "github-status-update"
 
 	githubUpdateTypeNewPatch              = "new-patch"
+	githubUpdateTypeMessage               = "send-message"
```

Comment:
Rather than call this just message, it seems like it's specifically encoding success + a message, can we call this `githubUpdateTypeSuccessMessage` or something?